pkgs/data/themes/arc/default.nix

```diff
@@ -43,8 +44,8 @@ stdenv.mkDerivation rec {
   '';
 
   configureFlags = [
+    "--with-cinnamon=${cinnamon.cinnamon-common.version}"
```

Comment:
This means gnome users of the theme will now have the cinnamon closure. Perhaps there should be split versions of a generic arc theme?

Reply:
Unsure, how much problems does this actually cause?

Reply:
> This means gnome users of the theme will now have the cinnamon closure. Perhaps there should be split versions of a generic arc theme?

I do not think so. It is just telling what version of cinnamon should be targeted. In order to be added to the closure it should cinnamon should be added to `buildInputs`, right?

Reply:
We're just pulling in version, so there's no way the whole thing gets added, I think